airbyte-webapp/src/views/layout/SideBar/SideBar.module.scss

```
@@ -1,6 +1,35 @@
 @use "../../../scss/colors";
 @use "../../../scss/variables";
 
+.navBar {
+  z-index: 9999;
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+  width: 100px;
+  min-width: 65px;
+  background-color: colors.$dark-blue;
+  padding: 23px 3px 15px 4px;
```

Same change as above
```suggestion
  padding: 23px 4px 15px 4px;
```

---

Fixed! Thanks